### DISCUSSION TITLE
Allow disabling of auto closing triple quotes

### DISCUSSION THREAD

**Discussion by huntfx** [microsoft/pylance-release#2649] [Enhancement]
I would appreciate more fine control over what is auto closed. My issue is that the auto closing on triple quotes is making me press backspace 6 times per commented out code, and I've tried living with it in the hopes I'll get used to it, but it's still quite annoying.

This is what I end up with each time I try and comment something out:

```python
''''''
# code I want to comment out
''''''
```

A solution recommended [here](https://github.com/microsoft/pylance-release/discussions/1984) was to disable auto closing on all quotes in all languages, which I've done for the time being, but it's not really a good solution.

-----------------------

As a more advanced solution, you could potentially check the line above. If the indent of that line is `n` (perhaps like a sphinx variable docstring) or `n-1` (normal docstring), then auto close, but for anything else then don't.

**Comment by judej** [comment]
This is request for an option to disable the auto quote feature.

As a quick workaround, you could use ctrl-Z to remove the auto close quotes.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
